docs/999-big-list-of-options.md

```diff
@@ -127,7 +127,8 @@ The file to write to. Will also be used to generate sourcemaps, if applicable. C
 
 #### output.format
 Type: `string`<br>
-CLI: `-f`/`--format <formatspecifier>`
+CLI: `-f`/`--format <formatspecifier>`<br>
+Default: `"cjs"`
```

Comment:
As a matter of fact, the default value is `"es"`, not `"cjs"`.

Reply:
Whoops ... testing it now, you're right. Perhaps my `rollup` was picking up config from elsewhere when I tested previously.